lexers/autoit.lua

```diff
@@ -2,26 +2,59 @@
 -- AutoIt LPeg lexer.
 -- Contributed by Jeff Stone.
 
-local lexer = require('lexer')
-local token, word_match = lexer.token, lexer.word_match
+local lexer = lexer
 local P, S = lpeg.P, lpeg.S
 
-local lex = lexer.new('autoit')
-
--- Whitespace.
-lex:add_rule('whitespace', token(lexer.WHITESPACE, lexer.space^1))
+local lex = lexer.new(...)
 
 -- Keywords.
-lex:add_rule('keyword', token(lexer.KEYWORD, word_match({
+lex:add_rule('keyword', lex:tag(lexer.KEYWORD, lex:word_match(lexer.KEYWORD, true)))
+
+-- Functions.
+lex:add_rule('function_builtin',
+  lex:tag(lexer.FUNCTION_BUILTIN, lex:word_match(lexer.FUNCTION_BUILTIN, true)) *
+    #(lexer.space^0 * P('(')^-1))
```

Comment:
`#(lexer.space^0 * P('(')^-1)` doesn't do anything here, so I would either:

* Remove the `^-1` if AutoIt allows variables/constants to use built-in function names. (Basically, '(' is required for this to be recognized as a built-in function.)
* Remove the entire expression if these words can only be interpreted as built-in functions and they don't require a trailing '(' to be used (like how functions are first-class values in Lua).

Reply:
Since both languages are very similar I ended up reusing the function rule from AutoHotkey here.